integrations/kuberay/prometheus_metadata.yaml

```diff
@@ -0,0 +1,96 @@
+        prometheus_name: ray_node_cpu_utilization
+        kind: GAUGE
+        value_type: DOUBLE
+      - name: prometheus.googleapis.com/ray_node_cpu_count/gauge
+        prometheus_name: ray_node_cpu_count
+        kind: GAUGE
+        value_type: DOUBLE
+      - name: prometheus.googleapis.com/ray_node_gpus_utilization/gauge
+        prometheus_name: ray_node_gpus_utilization
+        kind: GAUGE
+        value_type: DOUBLE
+      - name: prometheus.googleapis.com/ray_node_gpus_available/gauge
+        prometheus_name: ray_node_gpus_available
+        kind: GAUGE
+        value_type: DOUBLE
+      - name: prometheus.googleapis.com/ray_node_disk_usage/gauge
+        prometheus_name: ray_node_disk_usage
+        kind: GAUGE
+        value_type: DOUBLE
+      - name: prometheus.googleapis.com/ray_node_disk_free/gauge
+        prometheus_name: ray_node_disk_free
+        kind: GAUGE
+        value_type: DOUBLE
+      - name: prometheus.googleapis.com/ray_node_disk_io_write_speed/gauge
+        prometheus_name: ray_node_disk_io_write_speed
+        kind: GAUGE
+        value_type: DOUBLE
+      - name: prometheus.googleapis.com/ray_node_disk_io_read_speed/gauge
+        prometheus_name: ray_node_disk_io_read_speed
+        kind: GAUGE
+        value_type: DOUBLE
+      - name: prometheus.googleapis.com/ray_node_mem_used/gauge
+        prometheus_name: ray_node_mem_used
+        kind: GAUGE
+        value_type: DOUBLE
+      - name: prometheus.googleapis.com/ray_node_mem_total/gauge
+        prometheus_name: ray_node_mem_total
+        kind: GAUGE
+        value_type: DOUBLE
+      - name: prometheus.googleapis.com/ray_component_rss_mb/gauge
+        prometheus_name: ray_component_rss_mb
+        kind: CUMULATIVE
+        value_type: DOUBLE
+      - name: prometheus.googleapis.com/ray_node_mem_shared_bytes/gauge
+        prometheus_name: ray_node_mem_shared_bytes
+        kind: GAUGE
+        value_type: DOUBLE
+      - name: prometheus.googleapis.com/ray_component_cpu_percentage/gauge
+        prometheus_name: ray_component_cpu_percentage
+        kind: GAUGE
+        value_type: DOUBLE
+      - name: prometheus.googleapis.com/ray_node_gram_used/gauge
+        prometheus_name: ray_node_gram_used
+        kind: GAUGE
+        value_type: DOUBLE
+      - name: prometheus.googleapis.com/ray_node_gram_available/gauge
+        prometheus_name: ray_node_gram_available
+        kind: GAUGE
+        value_type: DOUBLE
+      - name: prometheus.googleapis.com/ray_node_network_receive_speed/gauge
+        prometheus_name: ray_node_network_receive_speed
+        kind: GAUGE
+        value_type: DOUBLE
+      - name: prometheus.googleapis.com/ray_node_network_send_speed/gauge
+        prometheus_name: ray_node_network_send_speed
+        kind: GAUGE
+        value_type: DOUBLE
+    install_documentation_url: https://cloud.google.com/kubernetes-engine/docs/add-on/ray-on-gke/how-to/enable-ray-on-gke
```

Comment:
It sounds like we actually have two docs:
1. https://cloud.google.com/kubernetes-engine/docs/add-on/ray-on-gke/how-to/enable-ray-on-gke for how to set up Kuberay
2. a doc in stackdriver for how to use GMP to scrape metrics for Kuberay (which is what `documentation.yaml` will produce).

This field populates this link in the UI:
![image](https://github.com/user-attachments/assets/9c33f5e8-215d-4e8a-b8d2-3c4318d0d8a5)

Which one do you want to link to? And should those docs 1 and 2 link to each other for discoverability as well?

Reply:
I'd like to promote the usage of the kuberay operator from GCP. If the operator is used, monitoring can be enabled to collect the metrics during the installation. If that's the case, do I still need the `documentation.yaml` file?

Reply:
The documentation.yaml page is used by the tech writers to create a doc in this directory: https://cloud.google.com/stackdriver/docs/managed-prometheus/exporters/introduction

For example, the [vLLM one](https://cloud.google.com/stackdriver/docs/managed-prometheus/exporters/vllm) in this directory assumes you already have set up vLLM, but want the information on how to observe it.

It might make sense to have docs in both places, especially for discoverability. But then you should work with the tech writers so that the docs link to each other.

Reply:
yes, I understand this part. However, Ray is a little different since we do have a supported operator for it. Anyhow, I will reach out to the tech writer. Thanks!